src/components/10-atoms/input-text/ui.test.js

```diff
@@ -86,3 +86,31 @@ test('should submit inputs correctly in form', async t => {
     .expect((await Selector('#password-id')).innerText)
     .eql(`password: ${password}`);
 });
+
+fixture('Input text - Max Length').page(
+  `${host}/iframe.html?id=atoms-input-text--input-text&knob-label*=&knob-name*=&knob-refid=&knob-placeholder=&knob-value=&knob-error=&knob-info=&knob-type=text&knob-maxlength=5&knob-counterMax=Character%20limit%20reached!&knob-checkmark=true`
+);
+
+test('should correctly show character count with counter within text', async t => {
+  const $axaTag = await Selector(() =>
+    document.querySelector('axa-input-text')
+  );
+  const $counterInfo = await $axaTag.find('.a-input-text__counter-info');
+  await t.expect($counterInfo.textContent).contains('Still 4 characters left');
+  const $checkMark = await $axaTag.find('.a-input-text__check');
+  await t.expect($checkMark.exists).ok();
+
+  const $input = await $axaTag.find(CLASS);
+  await t
+    .selectText($input)
+    .pressKey('delete')
```

Comment:
If I understand this correctly, you want to ensure the input field is initially empty, right? This and the line before can be implemented more elegantly by using the `{replace: true}` option as the 2nd arg. of `typeText`.

Reply:
See https://devexpress.github.io/testcafe/documentation/test-api/actions/action-options.html#typing-action-options

Reply:
This is not my code, but I can fix that. I was confused too.